test/hotspot/jtreg/serviceability/sa/TestJmapCore.java

```diff
@@ -73,6 +73,7 @@ public static void main(String[] args) throws Throwable {
     static void test(String type) throws Throwable {
         ProcessBuilder pb = ProcessTools.createTestJvm("-XX:+CreateCoredumpOnCrash",
                 "-Xmx512m", "-XX:MaxMetaspaceSize=64m", "-XX:+CrashOnOutOfMemoryError",
+                "-XX:-VerifyDependencies", "-XX:+IgnoreUnrecognizedVMOptions",
```

Comment:
Have you tried this test with `release`? I think `IgnoreUnrecognizedVMOptions` is positional, and should go before `VerifyDependencies` to actually work.

Reply:
I will try it and raise another commit if that is required

Reply:
It would make sense to change the order, even if it works right now. This would specify what option is being ignored more clearly.

Reply:
Hi Aleksey, release is working as well